aip/auth/4113.md

```diff
@@ -27,8 +33,8 @@ credentials.
 
 ### Credentials Generation
 
-gcloud default credentials can be generated via command ‘[gcloud auth
-application-default login][2]’.
+gcloud default credentials can be generated via command [gcloud auth
```

Comment:
We can have two sections here:

#### Gcloud User Credential

...

#### Impersonated Service Account

...